content/news/2022-11-12-bevy-0.9/index.md

```diff
@@ -0,0 +1,1509 @@
+
+To illustrate, consider the following UI:
+
+```txt
+root (green)
+  child1 (red)
+  child2 (blue)
+```
+
+By default these all have a z-index of 0. The root is at the bottom and each subsequent child stacks "on top":
+
+![z-index default](z-default.png)
+
+If we want the blue child to stack "behind" the earlier red child, we can set its z-index to a "local" value smaller than the default of 0:
+
+```rust
+blue.z_index = ZIndex::Local(-1);
+```
+
+![z-index blue local](z-blue-local.png)
+
+If we want the blue child to stack "behind" the green root, we can set its z-index to a "global" value smaller than the default of 0:
+
+```rust
+blue.z_index = ZIndex::Global(-1);
+```
+
+![z-index blue global](z-blue-global.png)
+
+Very useful stuff!
+
+[`ZIndex`]: https://docs.rs/bevy/0.9.0/bevy/ui/enum.ZIndex.html
+
+## Bevy UI Scaling
+
+<div class="release-feature-authors">authors: @Weibye</div>
+
+Bevy UI's global "pixel scale" can now be set using the [`UiScale`] resource:
+
+```rust
+// Render UI pixel units 2x bigger
+app.insert_resource(UiScale { scale: 2.0 })
+```
+
+This allows developers to expose arbitrary scale configuration to users in cases where that flexibility is beneficial.
+
+<video controls loop><source  src="ui_scaling.mp4" type="video/mp4"/></video>
+
+[`UiScale`]: https://docs.rs/bevy/0.9.0/bevy/ui/struct.UiScale.html
+
+## Audio Playback Toggling
+
+<div class="release-feature-authors">authors: @lovelymono</div>
+
+It is now possible to toggle audio playback, which will flip between playing and pausing:
+
+```rust
+// Old, manual toggling (still possible)
+if audio_sink.is_paused() {
+    audio_sink.play();
+} else {
+    audio_sink.pause();
+}
+
+// New, automatic toggling
+audio_sink.toggle();
+```
+
+## Time Scaling
+
+<div class="release-feature-authors">authors: @maniwani</div>
+
+The "global" time scale can now be configured on [`Time`], which scales the values common functions like `Time::delta_seconds()` return.
+
+```rust
+time.set_relative_speed(2.0);
+```
+
+In cases where unscaled values are required, you can use the new "raw" variants of these functions:
+
+```rust
+// The number of seconds elapsed since the last update, with time scaling taken into account.
+let delta = time.delta_seconds();
+
+// The number of seconds elapsed since the last update, with time scaling ignored.
+let raw_delta = time.raw_delta_seconds();
+```
+
+<video controls loop><source  src="time_scale.mp4" type="video/mp4"/></video>
+
+[`Time`]: https://docs.rs/bevy/0.9.0/bevy/time/struct.Time.html
+
+## Time Wrapping
+
+<div class="release-feature-authors">authors: @IceSentry</div>
+
+Some scenarios, [such as shaders](/news/bevy-0-9/#time-shader-globals), need to represent elapsed time values as `f32`, which will suffer from precision issues pretty quickly. To resolve this, [`Time`] has been extended to support "time wrapping":
```

Comment:
Might be interesting to link to [The Witness use case](http://the-witness.net/news/2022/02/a-shader-trick/) as a motivation and best practices.